src/commands/projects/get.ts

```diff
@@ -0,0 +1,15 @@
+import { fetchProjects } from '../../api/projects'
+import Base from '../base'
+
+export default class DetailedProjects extends Base {
+    static description = 'Retreive all projects in the current Organization'
+    static hidden = false
+    authRequired = true
+
+    public async run(): Promise<void> {
+        const projects = await fetchProjects(this.authToken)
+        return this.writer.showResults(projects.map((({ _id, key, name }) => (
+            { _id, key, name }
```

Comment:
we could include other properties like description, createdAt, updatedAt etc.

Reply:
should we just include all the properties?

Reply:
sure, if they're okay to return from the api, then they're fine to return from here 👍